impeller/entity/entity_pass.cc

```diff
@@ -198,12 +198,10 @@ std::optional<Rect> EntityPass::GetSubpassCoverage(
   std::shared_ptr<FilterContents> image_filter =
       subpass.delegate_->WithImageFilter(Rect(), subpass.xformation_);
 
-  // If the filter graph transforms the basis of the subpass, then its space
-  // has deviated too much from the parent pass to safely intersect with the
-  // pass coverage limit.
-  coverage_limit =
-      (image_filter && !image_filter->IsTranslationOnly() ? std::nullopt
-                                                          : coverage_limit);
+  // If the subpass has an image filter, then its coverage space may deviate
+  // from the parent pass and make intersecting with the pass coverage limit
+  // unsafe.
+  coverage_limit = image_filter ? std::nullopt : coverage_limit;
```

Comment:
Is this just punting on the coverage test if there is an image_filter?

Reply:
It just stops us from trying to use the screen area as a limit for coverage, since ImageFilters can wind up moving things around. We could make this more restrictive in the future by adding the ability to pull the transform from filter chains.

Reply:
> pull the transform from filter chains.

Yea, that's what I was thinking of when writing that doc.